docs/semgrep-supply-chain/policies.md

```diff
@@ -0,0 +1,74 @@
+---
+slug: policies
+title: Manage policies
+hide_title: true
+description: Use policies to define the conditions in which developers are notified of a finding or potentially blocked from merging their PR or MR.
+tags:
+  - Semgrep Supply Chain
+  - Semgrep AppSec Platform
+---
+
+# Manage policies (beta)
+
+Use Supply Chain policies to define the conditions in which developers are **notified** of a finding through a comment, or potentially **blocked** from merging their pull or merge request (PR or MR). This feature helps you manage noise and ensures that developers are only notified or potentially blocked based on the conditions you set.
+
+This feature enables you to configure the following:
+
+- **Scope**: These are the projects (repositories) that are affected by the policy.
+- **Conditions**: The conditions under which **actions** are performed. These conditions are typically related to the value of a finding's severity or reachability.
+- **Actions**: Actions that are performed on the defined scope when conditions are met.
+
+You can create as many policies as you need.
+
+## Feature maturity
+
+The Supply Chain policies feature is in **private beta**. To apply to this beta, reach out to [<i class="fa-regular fa-envelope"></i> support@semgrep.com](mailto:support@semgrep.com).
+
+## Prerequisites
+
+This feature requires the `semgrep:latest` Docker image or at least version 1.101.0 of the Semgrep CLI tool.
+
+## View your policies
+
+Only **admins** can view, create, edit, or delete policies.
+
+1. Sign in to [<i class="fas fa-external-link fa-xs"></i> Semgrep AppSec Platform](https://semgrep.dev/login).
+1. From the navigation menu, click **Rules** to expand the drop-down box, then click **Policies**.
+1. Click **Supply Chain**. This takes you to the Supply Chain policies tab.
+
+## Create a policy
+
+1. From the Supply Chain policies tab, Click **Create policy**.
+1. Provide a **Policy name**.
+1. Define the scope of the policy:
+  1. Click the drop-down box to select between **All Projects**, **Project**, or **Project tag**. 
+  1. For **Project** or **Project tag** values, a second drop-down box appears. Choose the **projects** or **project tags** to finish defining the scope. You can select more than one.
```

Comment:
I'm checking out the live preview and it seemed like these may have not been indented properly (though I'm going off of the live preview you linked in your slack message so it might be old). Currently seeing this in the live preview:
<img width="1034" alt="image" src="https://github.com/user-attachments/assets/252d61a1-5371-4d0b-b0f6-7d18d5bf6dc5" />

Reply:
Yup, looks like it requires a 4-space indent. Nice catch, thank you.